ccc/deliverydb.py

```diff
@@ -25,7 +26,7 @@ def sqlalchemy_with_db_cfg(
     )
 
 
-def psycopg_with_db_cfg(
+def psycopg_connection_with_db_cfg(
```

Comment:
nit: I think our convention at this point is to use the the word `from` instead of `with` like `psycopg_connection_from_db_cfg`